### DISCUSSION TITLE
Import/Export Lutris Library

### DISCUSSION THREAD

**Discussion by trymeouteh** [lutris/agora#112] [Ideas]
Please add a setting to import and export your Lutris library into a compressed zip file which can be imported again. This compressed file will contain all of your used runners and games and its data such as the banner image, game info, game options, runner options, all of it.

This way Lutris will have an official import/export feature to allow for gamers to export their library and back it up now and then and then be able to import it to their new gaming computers with just a few clicks.

This is to import/export the Lutris library, not the games themselves but the data to be able to reconstruct the Lutris library on another computer.

**Comment by strycore** [comment]
This should be fairly easy and will likely be added at some point.
The current dev version has the ability to import and export individual games (with their game files)
We also have plans to let users sync their lutris library to the website.
Since we tend to make the integration with the website optional, this means we'll have some kind of import / export feature not related to any external service.

**Reply by trymeouteh** [reply]
Playnite on Windows has this and this feature is what is keeping me from using Lutris. No point in adding my games and custom images into Lutris if I cannot backup the data.
